data-transfer/src/main/java/it/eng/datatransfer/exceptions/DataTransferExceptionAdvice.java

```diff
@@ -56,7 +56,7 @@ protected ResponseEntity<Object> handleTransferProcessExistsException(TransferPr
 				HttpStatus.BAD_REQUEST, request);
 	}
 	
-	@ExceptionHandler(value = { TransferProcessInvalidStateException.class })
+	@ExceptionHandler(value = { TransferProcessInvalidStateException.class, TransferProcessInvalidFormatException.class })
```

Comment:
Do we need specific exception, protocol and API since "all of them" are translated to either 400, 404 or 500?